pom.xml

```diff
@@ -1145,6 +1150,70 @@
           </exclusion>
         </exclusions>
       </dependency>
+      <!--
+        the AWS module pulls in jackson; its transitive dependencies can create
+        intra-jackson-module version problems.
+        -->
+      <dependency>
```

Comment:
OK, I'll do that. It's easily enough revisited in future.

The one thing that will probably need revisiting jackson dependency, as that's got the highest risk that someone will add it to another module. Though as it is driven by the same jackson.version variable as everywhere else, a duplicate declaration elsewhere isn't going to cause any harm